pandas/conftest.py

```diff
@@ -170,3 +170,11 @@ def string_dtype(request):
     * 'U'
     """
     return request.param
+
+
+@pytest.fixture(params=[0, 1], ids=lambda x: "axis {}".format(x))
```

Comment:
actually could also add ``'index', 'columns'`` here as well, *may* need to call this ``axis_frame``

Reply:
may as well add ``axis_series``

Reply:
I'm thinking that this fixture could replace all instances where there's a ``parametrize``. There are quite a few in the tests. Many of those don't support "index" and "columns" and it would give some failures (29 failures when I ran it with "index" and "columns").

I could add a ``axis_all`` fixture where params would be [0, 1, "index", "columns"]? Then people could differentiate between which one they need.